src/crate/crash/command.py

```diff
@@ -136,6 +137,9 @@ def _conf_or_default(key, value):
                         help='use -a to enable experimental auto-capitalization of SQL keywords')
     parser.add_argument('-U', '--username', type=str,
                         help='the username to authenticate in the database')
+    parser.add_argument('-W', '--password', action='store_true',
+                        dest='password', default=_conf_or_default('force_passwd_prompt', False),
```

Comment:
would it make sense to rename the `dest` parameter here already to `force_prompt_pwd` ? Reading the code further below I first thought this already contains the password